parm/gfs/oznHoriz.yaml

```diff
@@ -0,0 +1,216 @@
+      - {{DATA}}/ozn_data/horiz/ompsnp_npp.ges.{{ PDATEm6 | to_YMDH }}.ieee_d
+      - {{DATA}}/ozn_data/horiz/ompsnp_npp.ges.{{ PDATEm12 | to_YMDH }}.ieee_d
+      - {{DATA}}/ozn_data/horiz/ompsnp_npp.ges.{{ PDATEm18 | to_YMDH }}.ieee_d
+    levels: {{LEVELS}}
+    groups:
+      - name: GsiIeee
+        variables: &variables [obs]
+
+transforms:
+  - transform: select time
+    new name: horiz::GsiIeee::obs1
+    starting field: horiz::GsiIeee::obs
+    cycle: {{ PDATE | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::lat1
+    starting field: horiz::GsiIeee::lat
+    cycle: {{ PDATE | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::lon1
+    starting field: horiz::GsiIeee::lon
+    cycle: {{ PDATE | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::obs2
+    starting field: horiz::GsiIeee::obs
+    cycle: {{ PDATEm6 | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::lat2
+    starting field: horiz::GsiIeee::lat
+    cycle: {{ PDATEm6 | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::lon2
+    starting field: horiz::GsiIeee::lon
+    cycle: {{ PDATEm6 | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::obs3
+    starting field: horiz::GsiIeee::obs
+    cycle: {{ PDATEm12 | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::lat3
+    starting field: horiz::GsiIeee::lat
+    cycle: {{ PDATEm12 | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::lon3
+    starting field: horiz::GsiIeee::lon
+    cycle: {{ PDATEm12 | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::obs4
+    starting field: horiz::GsiIeee::obs
+    cycle: {{ PDATEm18 | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::lat4
+    starting field: horiz::GsiIeee::lat
+    cycle: {{ PDATEm18 | to_YMDH }}
+    for:
+      variable: [none]
+
+  - transform: select time
+    new name: horiz::GsiIeee::lon4
+    starting field: horiz::GsiIeee::lon
+    cycle: {{ PDATEm18 | to_YMDH }}
+    for:
+      variable: [none]
+
+graphics:
+
+  plotting_backend: Emcpy
+  figure_list:
+
+  # Map plots
+  # ---------
+
+    # Observations
+    - batch figure:
+        variables: ['obs1']
+        levels: {{LEVELS}}
+      dynamic options:
+        - type: vminvmaxcmap
+          level: ${level}
+          data variable: horiz::GsiIeee::${variable}
+
+      figure:
+        layout: [2,2]
+        figure size: [8,7]
+        title: "Ozone: Observed Values \n ompsnp_n20, \n level ${level}"
```

Comment:
Note to fill in variables for titles/filenames here too

Reply:
Noted and done.